src/main/java/org/elasticsearch/client/IndicesAdminClient.java

```diff
@@ -581,6 +581,21 @@
     AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String text);
 
     /**
+     * Analyze texts under the provided index.
+     *
+     * @param index The index name
+     * @param text  The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String... text);
```

Comment:
@javanna Yeah, my option is not good... 
 
I think Lee's comment is the following:
```
    AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String text);
    AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String... text);
    AnalyzeRequestBuilder prepareAnalyze(String text);
```
or 
```
    AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String... text);
    AnalyzeRequestBuilder prepareAnalyze(String text);
```

however, I think it is easy to understand that we have the other name of method, like
`prepareAnalyze` and `prepareAnalyzeForMultiValued`